manager/controllers/app/fybrik_notebook_read_flow_test.go

```diff
@@ -47,6 +53,47 @@ type ArrowRequest struct {
 	Columns []string `json:"columns,omitempty"`
 }
 
+func getPodLogs(pod *v1.Pod) string {
+	podLogOpts := v1.PodLogOptions{}
+	// creates the clientset
+	// can not use controller-runtime client. see https://github.com/kubernetes-sigs/controller-runtime/issues/452
+	clientset, err := kubernetes.NewForConfig(ctrl.GetConfigOrDie())
+	if err != nil {
+		return "error in getting access to K8S"
+	}
+	req := clientset.CoreV1().Pods(pod.Namespace).GetLogs(pod.Name, &podLogOpts)
+	podLogs, err := req.Stream(context.Background())
+	if err != nil {
+		return "error in opening stream"
+	}
+	defer podLogs.Close()
+	buf := new(bytes.Buffer)
+	_, err = io.Copy(buf, podLogs)
+	if err != nil {
+		return "error in copy information from podLogs to buf"
+	}
+	str := buf.String()
+	return str
+}
+
+func getPodsForSvc(svc *v1.Service, namespace string) (*v1.PodList, error) {
+	set := labels.Set(svc.Spec.Selector)
+	listOptions := metav1.ListOptions{LabelSelector: set.AsSelector().String()}
+	clientset, err := kubernetes.NewForConfig(ctrl.GetConfigOrDie())
+	if err != nil {
+		return nil, errors.New("error in getting access to K8S")
+	}
+	pods, err := clientset.CoreV1().Pods(namespace).List(context.Background(), listOptions)
+	if err != nil {
+		return nil, errors.New("error getting pods")
+	}
+	//nolint:rangeValCopy
+	for _, pod := range pods.Items {
+		fmt.Fprintf(os.Stdout, "pod name: %v\n", pod.Name)
```

Comment:
for i := range pods.Items {
     mt.Fprintf(os.Stdout, "pod name: %v\n", pods.Items[i].Name)
}

Reply:
tnx will fix that